src/components/responsive/ResponsiveContainer.js

```diff
@@ -5,6 +5,7 @@ import { Responsive } from 'semantic-ui-react';
 import { connect } from 'react-redux';
 import MobileView from './MobileView';
 import DesktopView from './DesktopView';
+import { booleanLiteral } from '@babel/types';
```

Comment:
why is this needed?

Reply:
Great question! Totally missed that it was automatically added. Will fix.